hub/scripts.js

```diff
@@ -397,6 +397,39 @@
     });
   }
 
+  async function decoratePromotions() {
+    if (document.querySelector('main .promotion') instanceof HTMLElement) {
+      // A promotion has already been defined on the page.
+      // Do not inject another one.
+      return;
+    }
+
+    let promotionName = document.querySelector('head meta[name="promotion"]');
+    if (!promotionName) {
+      return;
+    }
+    promotionName = promotionName.getAttribute('content').toLowerCase();
+
+    try {
+      const { locale } = window.fedPub;
+      const promotion = await window.fetch(`${locale === 'en' ? '' : `/${locale}`}/promotions/hub/${promotionName}.plain.html`);
```

Comment:
```suggestion
      const response = await window.fetch(`${locale === 'en' ? '' : `/${locale}`}/promotions/hub/${promotionName}.plain.html`);
```

Naming the "response" object "promotion" is confusing I think.